supervisor/docker/__init__.py

```diff
@@ -230,3 +230,17 @@ def repair(self) -> None:
             _LOGGER.debug("Networks prune: %s", output)
         except docker.errors.APIError as err:
             _LOGGER.warning("Error for networks prune: %s", err)
+
+    def check_denylist_images(self) -> bool:
+        """Return a boolen if the host has images in the denylist."""
```

Comment:
Sorry, I missed this one:
boolean

Reply:
```suggestion
        """Return a boolean if the host has images in the denylist."""
```

Google thinks it should be "deny list" or "deny-list". "denylist" seems quite frequently used too though.

Reply:
I was going by <https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=49decddd39e5f6132ccd7d9fdc3d7c470b0061bb>